ee/vellum_ee/workflows/display/tests/workflow_serialization/generic_nodes/test_adornments_serialization.py

```diff
@@ -1,47 +1,55 @@
-import pytest
+from uuid import uuid4
 
 from deepdiff import DeepDiff
 
 from vellum.workflows.inputs.base import BaseInputs
 from vellum.workflows.nodes.bases.base import BaseNode
-from vellum.workflows.nodes.core.map_node.node import MapNode
+from vellum.workflows.nodes.core.retry_node.node import RetryNode
 from vellum.workflows.outputs.base import BaseOutputs
-from vellum.workflows.state.base import BaseState
+from vellum_ee.workflows.display.base import WorkflowInputsDisplay
+from vellum_ee.workflows.display.nodes.vellum.base_node import BaseNodeDisplay
 
 
 class Inputs(BaseInputs):
-    foo: str
+    input: str
 
 
-class State(BaseState):
-    bar: str
+@RetryNode.wrap(max_attempts=3)
+class InnerRetryGenericNode(BaseNode):
+    input = Inputs.input
 
+    class Outputs(BaseOutputs):
+        output: str
 
-@MapNode.wrap(items=[1, 2, 3])
-class MapGenericNode(BaseNode):
-    item = MapNode.SubworkflowInputs.item
-    foo = Inputs.foo
-    bar = State.bar
 
-    class Outputs(BaseOutputs):
-        value: str
+class InnerRetryGenericNodeDisplay(BaseNodeDisplay[InnerRetryGenericNode.__wrapped_node__]):  # type: ignore
```

Comment:
`__wrapped_node__` raises `name-defined` 
I can't do `type: ignore[name-defined]` because it makes the line too long. How do I add both type ignore and noqa in the same line?

Reply:
On formatting:
- I think `# type: ignore noqa: E501` should work

However:
- I would take a look at the [wrap](https://github.com/vellum-ai/vellum-python-sdks/blob/main/ee/vellum_ee/workflows/display/nodes/vellum/try_node.py#L78) pattern we established for Try Node Displays (example usage [here](https://github.com/vellum-ai/vellum-python-sdks/blob/de4a33c3c7b84d9e5e1f46ce4922403d2efa2d32/ee/codegen_integration/fixtures/simple_node_with_try_wrap/code/display/nodes/prompt_node.py#L10-L11)). It allows us to keep `__wrapped_node__` private from user land and shifts some of the adorning logic from BaseNodeDisplay up to the relevant adornment node